raiden/api/rest.py

```diff
@@ -20,13 +20,10 @@
 from web3 import Web3
 from webargs.flaskparser import parser
 
-from raiden.api.objects import (
-    AddressList,
-    PartnersPerTokenList,
-    DashboardGeneralItem,
-    DashboardGraphItem,
-    DashboardTableItem
-)
+from raiden.api.objects import AddressList, PartnersPerTokenList
```

Comment:
@JONAF2103 would you be so kind to check the changes here? i believe they stem from one of your PRs and i'm not so sure about their correctness

thanks

Reply:
is this comment related with the imports? 

seems just like a reorg right?

Reply:
Sure

Reply:
> is this comment related with the imports?
> 
> seems just like a reorg right?

i meant in the entire module. sorry i wasn't clear enough